$NetBSD: patch-ab,v 1.20 2014/05/29 07:57:07 adam Exp $

* XXX
* Tell perl that Haiku need haikuish.h installed as well

--- installperl.orig	2015-05-13 20:19:29.000000000 +0000
+++ installperl
@@ -327,7 +327,6 @@ $do_installprivlib = 0 if $versiononly &
 mkpath($installprivlib);
 mkpath($installarchlib);
 mkpath($installsitelib, $opts{verbose}, 0777) if ($installsitelib);
-mkpath($installsitearch, $opts{verbose}, 0777) if ($installsitearch);
 
 if (-d 'lib') {
     find({no_chdir => 1, wanted => \&installlib}, 'lib')
@@ -371,6 +370,10 @@ elsif ($Is_Cygwin) { # On Cygwin symlink
 
     # AIX needs perl.exp installed as well.
     push(@corefiles,'perl.exp') if $^O eq 'aix';
+    if ($^O eq 'haiku') {
+        mkpath("$installarchlib/CORE/haiku");
+        push(@corefiles, 'haiku/haikuish.h');
+    }
 }
 foreach my $file (@corefiles) {
     # HP-UX (at least) needs to maintain execute permissions
